kinesis_video_streamer/test/kinesis_video_streamer_test_utils.h

```diff
@@ -20,7 +20,6 @@
 #include <kinesis_video_streamer/ros_stream_subscription_installer.h>
 #include <kinesis_video_streamer/streamer.h>
 
-using namespace std;
 using namespace com::amazonaws::kinesis::video;
```

Comment:
Shouldn't lines 23-25 also be removed as well?

Reply:
Yeah probably but they require much more verbose changes and have a much smaller chance of problem, I preferred to do this incremental thing